lesson12/index.js

```diff
@@ -0,0 +1,68 @@
+const ul = document.getElementById('js-ul');
+const div = document.getElementById('js-wrapper');
+const button = document.getElementById("js-button");
+
+
+function addLoading() {
+  const loading = document.createElement('img');
+  loading.src = "./loading-circle.gif";
+  loading.id = "loading";
+  ul.appendChild(loading);
+}
+
+function removeLoading() {
+  const loading = document.getElementById('loading');
+  loading.remove();
+}
+
+
+async function fetchData() {
+  addLoading();
+  const response = await fetch("https://jsondata.okiba.me/v1/json/TOqvy211002104750");
+  if (!response) {
+    throw new Error(response.statusText);
+  } else {
+    const json = await response.json();
+    return json;
+  }
+}
+
+async function createLists(responseData) {
+  removeLoading();
```

Comment:
【nits】
How about putting this function together in a process of createList only and executing removeLoading elsewhere?

Reply:
Thanks for the review!!

I figured it wasn't a good idea for one function to do the work of another.
I fixed that.